be/src/vec/functions/function.h

```diff
@@ -395,6 +404,14 @@ class IFunction : public std::enable_shared_from_this<IFunction>,
         return Status::OK();
     }
 
+    Status eval_inverted_index(FunctionContext* context,
+                               const vectorized::NameAndTypePair& data_type_with_name,
+                               segment_v2::InvertedIndexIterator* iter, uint32_t num_rows,
+                               roaring::Roaring* bitmap) const override {
+        LOG(FATAL) << "eval_inverted_index is not implemented for IFunction";
```

Comment:
Why not return ERROR

Reply:
just same with other functions in this base class.